README.md

```diff
@@ -91,7 +90,8 @@ You should now have a folder with the `SITE_NAME` you provided to the above
 script with the basics completed for you.
 
 On your platform of choice [GitHub], [GitLab], etc. Create a new Git repository
-for your new site.
+for your new site. (Not necessary for a development instance unless you want to
+save changes to it.) 
```

Comment:
Is this accurate? I feel quite silly going through all this "institution" stuff to spin up a temporary dev sandbox.

Reply:
I guess, rather, is it "recommended" that a git repo and remote be created for development projects? What changes might get saved here?